jiant/models.py

```diff
@@ -1050,22 +1013,23 @@ def _pair_sentence_forward(self, batch, task, predict):
                 logits = classifier(sent1, sent2, mask1, mask2, [batch["idx1"]], [batch["idx2"]])
             else:
                 logits = classifier(sent1, sent2, mask1, mask2)
-        out["logits"] = logits
+        logits_np = logits
```

Comment:
Is this var a numpy array at this point in the code? If it is, let's rename the `logits` var used above this line to consistently show it's a numpy array, if it isn't let's remove the `*_np`.